### PR TITLE
Feature add optimism goerli network

### PR DESCRIPTION
**Pull request by CarlosQ96**
Related issue: https://github.com/Giveth/impact-graph/issues/1036

### CODE REVIEW

File: migration/1687383705794-AddOptimismGoerliTokens.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.manager.save(
+      Token,
+      seedTokens
+        .filter(token => token.networkId === NETWORK_IDS.OPTIMISM_GOERLI)
+        .map(t => {
+          t.address = t.address?.toLowerCase();
+          return t;
+        }),
+    );
```
Comment (mohammadranjbarz) [inline migration/1687383705794-AddOptimismGoerliTokens.ts:25]: I think you need to add a condition to not add these test tokens in production env

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]
Thanks @CarlosQ96 , except comment that I put in migrations file other things seems good to me

**Comment by CarlosQ96** [comment]
Thanks Mohammad addded the condition to not add them in production

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @CarlosQ96  LGTM

**Pull request merged**